### PR TITLE
Add home key in yaml and typo fix in burrow/README.md

### PR DESCRIPTION
**Pull request by AdamDang**
Line 39: tables lists -> table lists

### CODE REVIEW

**Comment by AdamDang** [comment]
tests failed on CircleCI, Close this PR first.

**Comment by AdamDang** [comment]
Add home key:
 home: https://burrow.com/

**Comment by AdamDang** [comment]
@timstoop

File: stable/burrow/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,8 @@
 name: burrow
-version: 0.4.2
+version: 0.4.3
 appVersion: 0.17.1
 description: Burrow is a permissionable smart contract machine
+home: https://burrow.com/
```
Comment (timstoop) [inline stable/burrow/Chart.yaml]: This site is about a luxury couch...

Comment (AdamDang) [reply stable/burrow/Chart.yaml]: Hi @timstoop , I modified home key with "https://github.com/hyperledger/burrow"
This is from the README.md of burrow.

**Pull request merged**